nanshe_workflow/dask_config.yaml

```diff
@@ -0,0 +1,55 @@
+logging:
+  distributed: info
+  distributed.client: warning
+  bokeh: critical
+  # http://stackoverflow.com/questions/21234772/python-tornado-disable-logging-to-stderr
+  tornado: critical
+  tornado.application: error
+
+compression: auto
+
+# Scheduler options
+bandwidth: 100000000    # 100 MB/s estimated worker-worker bandwidth
+allowed-failures: 3     # number of retries before a task is considered bad
+pdb-on-err: False       # enter debug mode on scheduling error
+transition-log-length: 100000
+work-stealing: True     # workers should steal tasks from each other
+
+# Worker options
+multiprocessing-method: spawn
```

Comment:
This is absolutely essential to avoid segmentation faults on the workers during dictionary learning. OpenBLAS seems to not respond well to `forkserver`, which is the default option.